documentation/src/main/antora/modules/ROOT/pages/tutorial-create-my-first-component.adoc

```diff
@@ -0,0 +1,165 @@
+= Creating your first component
+:page-partial:
+
+This tutorial walks you through all the required steps to get started with *Talend Component Kit*, from the creation of a simple component to its integration into *Talend Studio*.
+
+The component created in this tutorial is a simple output component that only reads the data it receives and displays it in the logs, along with an extra information entered by a user.
+
+Once the prerequisites completed, this tutorial should take you about 20 minutes.
+
+image::tutorial_build_job_run.png[Job run]
+
+== Prerequisites
+As this tutorial aims at helping you to create your very first component, it also covers some of the installation steps that are specific to Talend Component Kit to get your development environment ready. But first, make sure to:
+
+- Download and install a Java JDK 1.8 or greater.
+// Download and install Maven 3.5.x.
+- Download and install Talend Studio. For example, from link:https://sourceforge.net/projects/talend-studio[Sourceforge].
+- Download and install https://www.jetbrains.com/idea/download[IntelliJ].
+- Download the Talend Component Kit plugin for IntelliJ. The detailed installation steps for the plugin are available in xref:installing-talend-intellij-plugin.adoc[this document].
+
+== Generating a simple component project
+The first step in this tutorial is to generate a project containing a simple output component using the starter.
```

Comment:
using the starter ==> here we will use the *Talend Component Kit plugin for Intellij*